### PR TITLE
Feature create ui user in domain

### PR DESCRIPTION
**Pull request by nguyenhuy95dn**

### CODE REVIEW

File: app/src/main/java/com/framgia/forder/screen/userindomain/ItemUserInDomainViewModel.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    public String getAvatar() {
+        if (mUser.getAvatar() != null) {
+            return mUser.getAvatar().getUrl();
+        }
+        return "";
+    }
+
+    public String getName(){
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/userindomain/ItemUserInDomainViewModel.java]: ERROR: '{' is not preceded with whitespace.

File: app/src/main/java/com/framgia/forder/screen/userindomain/ItemUserInDomainViewModel.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        return mUser.getName();
+    }
+
+    public String getEmail(){
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/userindomain/ItemUserInDomainViewModel.java]: ERROR: '{' is not preceded with whitespace.

File: app/src/main/java/com/framgia/forder/screen/userindomain/UserInDomainPresenter.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ * Listens to user actions from the UI ({@link UserInDomainFragment}), retrieves the data and
+ * updates
+ * the UI as required.
+ */
+final class UserInDomainPresenter implements UserInDomainContract.Presenter {
+    private static final String TAG = UserInDomainPresenter.class.getName();
+
+    private final UserInDomainContract.ViewModel mViewModel;
+
+    public UserInDomainPresenter(UserInDomainContract.ViewModel viewModel) {
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/userindomain/UserInDomainPresenter.java]: ERROR: Redundant 'public' modifier.

File: app/src/main/res/menu/menu_user_in_domain.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu
```
Comment (daolq3012) [inline app/src/main/res/menu/menu_user_in_domain.xml:2]: WARNING: [UnusedResources] The resource `R.menu.menu_user_in_domain` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -180,4 +180,7 @@
     <string name="manager_in_shop">Manager in shop</string>
     <string name="close">Close</string>
     <string name="see_manager">See Manager</string>
+    <string name="user_in_domain">User in domain</string>
+    <string name="delete">Delete</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:184]: WARNING: [UnusedResources] The resource `R.string.delete` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -180,4 +180,7 @@
...
+    <string name="set_manager">Set Manager</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:185]: WARNING: [UnusedResources] The resource `R.string.set_manager` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/menu/menu_user_in_domain.xml (hunk 4 above)
Comment (nguyenhuy95dn) [reply app/src/main/res/menu/menu_user_in_domain.xml:2]: use at next pull

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (nguyenhuy95dn) [reply app/src/main/res/values/strings.xml:184]: as above

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (nguyenhuy95dn) [reply app/src/main/res/values/strings.xml:185]: as above

**Pull request merged**